src/main/main.scss

```diff
@@ -0,0 +1,111 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```

Comment:
The variables should be in `variables.scss` file. Don't copy it to your file, just merge the change from `develop` to your branch. Besides, your `scss` file should be in `src/index.scss` file.